person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative 'nameable'
+
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- Please create the `has-many` side of `Person` and `Rental` (a person has many rentals).
- A method for adding rentals should be present 👍🤓
- According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md), The person itself should be sent as a parameter to create the new rental achieving the `has-many` association

```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
    @rentals = []
  end

  def add_rental(...)
  end

  .
  .
end
```

Reply:
Thank you for bringing this to my attention. I am updating the code 🙂.